src/darwin/Framework/CHIP/MTRDeviceController_Concrete.h

```diff
@@ -223,6 +223,11 @@ NS_ASSUME_NONNULL_BEGIN
                             queue:(dispatch_queue_t)queue
                        completion:(void (^)(NSURL * _Nullable url, NSError * _Nullable error))completion;
 
+/**
+ * Returns YES if the MTRDevice corrresponding to the given node ID is a thread device, NO otherwise.
```

Comment:
```suggestion
 * Returns YES if the MTRDevice corrresponding to the given node ID is known to be a thread device, NO otherwise.
```
because maybe it's using Thread but _we don't know that yet_.